hack/deploy-local.sh

```diff
@@ -30,13 +30,13 @@ fi
 kubectl apply -f ./ServiceAccount.yaml
 kubectl apply -f ./Identity.yaml
 kubectl apply -f ./Service.yaml
-kubectl apply -k .
+kubectl apply -k ./kustomizations/local/istanbul
```

Comment:
I had to copy the same `Deployment.yaml` several times; 
it’s good enough for now… we can refactor it later.